reflex/utils/processes.py

```diff
@@ -45,7 +45,7 @@ def get_api_port() -> int:
     """
     port = urlparse(get_config().api_url).port
     if port is None:
-        port = urlparse(constants.API_URL).port
+        port = 80
```

Comment:
```suggestion
        # use the default HTTP port in the absence of any specific port on `api_url`
        port = 80
```
first read i definitely thought this should have been `8000`, but second read i realized if the `urlparse` does not find a port, then we use the default HTTP port.